doto-frontend/src/components/pages/Calendar/CalendarListView.js

```diff
@@ -4,19 +4,32 @@ import Radio from "@material-ui/core/Radio";
 import "./Calendar.css";
 
 const CalendarListView = ({ tasks }) => {
+    const today = new Date();
```

Comment:
Maybe in the future we could look into using a standardised date library. Right now its fine.